module/helpers/hand.mjs

```diff
@@ -0,0 +1,421 @@
+/* eslint-disable no-console */
+import { aCards, mCardMap } from './cards.mjs';
+import { Deck } from './deck.mjs';
+
+export class Hand {
+  constructor(isHostile) {
+    this.isHostile = !!isHostile;
+
+    this.cards = [];
+    this.discards = [];
+    this.rjoker = false;
+    this.bjoker = false;
+    this.held = -1;
+    this.override = -1;
+    this.initiative = -1;
+  }
+
+  static fromObject(proto) {
+    const isHostile =
+      typeof proto !== 'undefined' &&
+      typeof proto.isHostile !== 'undefined' &&
+      typeof proto.isHostile === 'boolean' &&
+      proto.isHostile;
```

Comment:
There are probably a few areas we could make fancy improvements, but this is an obvious one 😂 
```suggestion
    const isHostile = Boolean(proto?.isHostile);
```

Reply:
Yes, if you look at the commit called tidy up flag access you can see me tidy up lots of instances of that kind of thing.